lib/vue/src/components/resourceList/CResourceList.vue

```diff
@@ -70,6 +70,37 @@
         @sort-changed="pagination.page = 1"
         @row-clicked="$emit('row-clicked', $event)"
       >
+      <template #head()="field">
+        <div class="d-flex align-items-center">
+          <div
+            class="d-flex align-self-center text-nowrap"
+          >
+            {{ field.label }}
+          </div>
+
+          <b-button
+            v-if="field.field.sortable"
+            variant="outline-extra-light"
+            class="d-flex align-items-center text-secondary d-print-none border-0 px-1 ml-1"
+            @click="handleSort(field)"
+            >
+            <font-awesome-layers
+              class="d-print-none"
+            >
+              <font-awesome-icon
+                :icon="['fas', 'angle-up']"
+                class="mb-1 text-grey"
```

Comment:
One really small thing here, if you can remove text-grey, since it doesn't do anything anymore
Same for the one in CRecordList